### PR TITLE
Fix dokku certs:add file input bug

### PR DESCRIPTION
**Pull request by u2mejc**
- fixes issue #1658
- add better output for argument files missing
- don't delete (move) the certs passed in

### CODE REVIEW

File: plugins/certs/commands [hunk 1]
```diff
@@ -55,8 +61,8 @@ certs_set() {
   fi
 
   mkdir -p "$DOKKU_ROOT/$APP/tls"
-  mv "$CRT_FILE" "$DOKKU_ROOT/$APP/tls/server.crt"
-  mv "$KEY_FILE" "$DOKKU_ROOT/$APP/tls/server.key"
+  cp "$CRT_FILE" "$DOKKU_ROOT/$APP/tls/server.crt"
+  cp "$KEY_FILE" "$DOKKU_ROOT/$APP/tls/server.key"
```
Comment (michaelshobbs) [inline plugins/certs/commands]: i suppose it doesn't matter as `$TEMP_DIR` get removed but why change to `cp`?

**Comment by michaelshobbs** [comment]
I'm not certain this fixes #1658. This plugin could definitely use some tests anyway, though. Care to take a stab at that and include the case from #1658? You can use the test certs in `tests/server_ssl*.tar`.

File: plugins/certs/commands (hunk 1 above)
Comment (u2mejc) [inline plugins/certs/commands]: @michaelshobbs no you tell me, why you delete my certs bro? That was my only copy!!! :wink:

**Comment by michaelshobbs** [comment]
The way I read it, we are working in a temp folder and not on the input. Maybe I misread it?

**Comment by u2mejc** [comment]
ah sorry I understand your confusion now, github isn't showing you the whole file. The temp dir is for the tar file and unrelated to passing in file arguments from the command line. 

Here's an example:
dokku certs:add ruby-sample /home/myuser/server.crt /home/myuser/server.key
CRT_FILE=$3 

If you *mv* $CRT_FILE, your literally deleting the user's /home/myuser/server.crt. The verbiage add doesn't imply delete to the user and I can't think of a single program I would expect to delete my key and force me to retrieve a new copy from the CA. Does heroku delete the keys from your laptop when it copies to the server? I can see where that might be confusing.

**Comment by michaelshobbs** [comment]
No I think the Heroku client just copies them up to the paas. I figured I just misread it. Still would be great to get at least one test written for each input type. i.e. file and tarball. 

Care to take a stab at that?

**Comment by u2mejc** [comment]
It's 5:59 on friday, happy to! :+1:

**Comment by michaelshobbs** [comment]
I definitely did not mean right this moment sir. 😄️

**Comment by hhff** [comment]
thanks duuuuuudes

**Comment by josegonzalez** [comment]
@u2mejc *bump* I also don't see how it solves the bug.

**Comment by u2mejc** [comment]
Bummer, @michaelshobbs @josegonzalez when you cloned and tested it, what was the results of your tests?

**Comment by josegonzalez** [comment]
I didn't try it, I'm just saying it doesn't *look* like it does.

File: plugins/certs/commands [hunk 2]
```diff
@@ -11,11 +11,17 @@ is_tar_import() {
 }
 
 is_file_import() {
-  local CRT_FILE="$3"
-  local KEY_FILE="$4"
-
-  if [[ -f $CRT_FILE ]] && [[ -f $KEY_FILE ]]; then
-    return 0
+  local CRT_FILE="$1"
+  local KEY_FILE="$2"
```
Comment (michaelshobbs) [inline plugins/certs/commands]: i see the fix now!!!

**Comment by michaelshobbs** [comment]
@josegonzalez no tests!! why u merge bro!? :smile:

**Comment by u2mejc** [comment]
Sorry I didn't have a chance to respond sooner! I'll work on tests and submit a separate PR.

**Comment by josegonzalez** [comment]
@michaelshobbs I'm too trigger happy, sorry :(

**Comment by josegonzalez** [comment]
I saw you say you understood it so I was like LG2ME

**Comment by michaelshobbs** [comment]
I was just so excited I finally saw the feex 😜️

**Pull request merged**